introduction.md

```diff
@@ -6,3 +6,8 @@ Hi, I'm "drakpappa".
 
 Some of my favorite Code School paths are: Git, Ruby.
 
+From my kill list: 
+* Git
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?